### PR TITLE
Edmundson Preface

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: journal/modernized/01-preface.adoc [hunk 1]
```diff
@@ -374,7 +375,8 @@ the gift of grace bestowed on you through Christ Jesus,
 that your latter end may be peace, and your future state eternal happiness.
 So in Christian love remain your well-wishing friend,
 
-John Stoddart.Dublin, the 26th of the Eighth month, 1714.
```
Comment (maijapapaya) [inline journal/modernized/01-preface.adoc:377]: So, these were my best guesses at how this should be formatted, based upon what I've seen in other documents, and in the style guide. Will it be more editing stuff and trying to figure it out and make it work like this with the intakes??....I was feeling like I was changing it too much.

Comment (jaredh159) [reply journal/modernized/01-preface.adoc:377]: Yup, this is perfect, a signature followed by a signed-section-context-close, that's exactly right.  It seems fuzzy at first, but you'll start recognizing the patterns and you'll feel more confident the more you work on this type of stuff.

File: journal/modernized/01-preface.adoc [hunk 2]
```diff
@@ -1,4 +1,5 @@
-== Preface.By John Stoddard
+== Preface. By John Stoddard
+//^^ Does his name need to be here since it's signed below? I also noticed it's omitted in the original edition.
```
Comment (jaredh159) [inline journal/modernized/01-preface.adoc]: Yup, this is a little weird, I think we should probably delete the name here.

File: journal/modernized/01-preface.adoc [hunk 3]
```diff
@@ -1,4 +1,5 @@
...
 
 Friendly Reader,
```
Comment (jaredh159) [inline journal/modernized/01-preface.adoc:4]: Here's one thing you missed -- this should be marked as a `[.salutation]` and also in the other edition. 👍

**Review by jaredh159 — CHANGES_REQUESTED** [review]
Maija, sorry for the delay getting this reviewed.  Looks pretty good, I know it's a little overwhelming at first, but you're doing great! Couple changes here and then we'll merge. :)

File: journal/modernized/01-preface.adoc [hunk 4]
```diff
@@ -74,7 +74,7 @@ and openly triumphed over principalities and powers.
 Now the old covenant was to pass away, and the new covenant to be established,
 and the priesthood changed, by the great High Priest without sin,
 and higher than the heavens; made not after the law of a carnal commandment,
-but by the power of an endless life, a priest forever, after the order of Melchisedec.
+but by the power of an endless life, a priest forever, after the order of Melchizedek.
```
Comment (maijapapaya) [inline journal/modernized/01-preface.adoc:77]: It seems this is an older rendering of Melchizedek, or an English one...I changed it here but left it in the original. Also, I forgot I have to add the proofread changes while I"m doing this. Glad I remembered this early on in the book!! :)

**Review by jaredh159 — APPROVED** [review]

**Pull request merged**